src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj

```diff
@@ -39,7 +39,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.61701" />
     <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="$(MicroBuildCoreVersion)" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20104.2" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20158.1" />
```

Comment:
Is updating this version necessary? I'd rather not do that unless you've done testing on this exe to ensure nothing has regressed.

Reply:
I can revert this. I just went with the newer version because it is the most downloaded in NuGet, and I'm trying to troubleshoot this problem.